static/sass/ds/6-components/_message.scss

```diff
@@ -22,7 +23,7 @@
     border: 1px solid $color-white-off;
     border-radius: 50%;
     cursor: pointer;
-    padding: .25rem .45rem;
+    padding: .5rem .5rem;
```

Comment:
For some reason this padding setting is looking a little oblong on my screen. I always set a width and height on circle buttons, but maybe that's old school 👵. If you go that route, `l-align-center-children` is helpful to center the little x icon.

Reply:
@ashley-hebler Yeah, I jacked that up. Should be fixed now.